project/JavaFormatter.scala

```diff
@@ -24,7 +23,8 @@ object JavaFormatter extends AutoPlugin with ProjectFileIgnoreSupport {
   override def projectSettings: Seq[Def.Setting[_]] = Seq(
     //below is for sbt java formatter
     (excludeFilter in format) := {
-      val simpleFileFilter = new SimpleFileFilter(file => isIgnoredByFileOrPackages(file))
+      val ignoreSupport = new ProjectFileIgnoreSupport((baseDirectory in ThisBuild).value / ignoreConfigFileName, descriptor)
```

Comment:
Here is the fix. It's using

```scala
(baseDirectory in ThisBuild).value / ignoreConfigFileName
```

instead of

```scala
file(ignoreConfigFileName)
```

(And another time in ScalaFixSupport below)